pkg/gameservers/controller.go

```diff
@@ -498,6 +498,27 @@ func (c *Controller) addGameServerHealthCheck(gs *v1alpha1.GameServer, pod *core
 	}
 }
 
+// applyGameServerAddressAndPortIfNeeded calls applyGameServerAddressAndPort if
+// the address and port is not already applied in game server status
+// NOTE THIS IS NOT PART OF AUTO SCALER
+func (c *Controller) applyGameServerAddressAndPortIfNeeded(gs *v1alpha1.GameServer) (*v1alpha1.GameServer, bool, error) {
```

Comment:
Is this from Mark's fix? If not, might be better to remove this and rebase.

Reply:
`// NOTE THIS IS NOT PART OF AUTO SCALER` :smile:

Reply:
ready to rebase on master